Meshtastic-iOS.xcodeproj/project.pbxproj

```diff
@@ -479,6 +491,11 @@
 			package = 0DEE520725D0BA71003C6EB7 /* XCRemoteSwiftPackageReference "swift-protobuf" */;
 			productName = SwiftProtobuf;
 		};
+		8A713CCF2623EAD300F6D4BB /* MessageKit */ = {
```

Comment:
If MessageKit is not used can you please remove it

Reply:
What's the proper way to do this?